query_db.py

```
@@ -0,0 +1,55 @@
+        
+        # Get column names
+        columns = [description[0] for description in cursor.description]
+        
+        # Print results in a nice table format
+        print("\nResults:")
+        print(tabulate(rows, headers=columns, tablefmt="grid"))
+        print(f"\nTotal rows: {len(rows)}")
+            
+    except sqlite3.Error as e:
+        print(f"\nDatabase Error: {e}")
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)